Sources/Swift/SentryOptionsValidator.swift

```diff
@@ -0,0 +1,37 @@
+@objcMembers
+class SentryOptionsValidator: NSObject {
+    @objc
+    static func validate(options: Options) {
+        SentryLog.debug("Validating Sentry SDK options")
+        if !Self.isCacheDirectoryPathValid(path: options.cacheDirectoryPath) {
+            SentryLog.fatal("The configured cache directory path looks invalid, the SDK might not be able to write reports to disk: \(options.cacheDirectoryPath)")
```

Comment:
`m`: The log message should indicate how you can fix the error. Now, it only states it's not working, but instead,d it should communicate that the path is too long and how to fix it.

Reply:
I agree, I'll change that to provide actual feedback how to solve the issue